geode-core/src/main/java/org/apache/geode/internal/security/IntegratedSecurityService.java

```diff
@@ -168,6 +167,19 @@ public void logout() {
     ThreadContext.remove();
   }
 
+  public void logoutJmxUser(String user, Subject currentUser) {
+    try {
+      logger.debug("Logging out " + currentUser.getPrincipal());
+      currentUser.logout();
+    } catch (ShiroException e) {
+      logger.info("error logging out: " + currentUser.getPrincipal());
```

Comment:
Shouldn't this be `logger.error` or, at least, `logger.warn`?.

Reply:
I try to be consistent like it was written in methods before(login, logout) so I keep this with _logger.info_. If you think that this should be with _error_ or _warn_,  I can change it in all methods.

Reply:
Good point.
I believe it's better to log these exceptions using `warn` or `error` level so users can actually notice the failure, but I might be missing something. @jinmeiliao surely has more context about why these exceptions were always logged as `info`.